app/assets/javascripts/darkswarm/directives/body_scroll.js.coffee

```diff
@@ -4,6 +4,12 @@ Darkswarm.directive "bodyScroll", ($rootScope, BodyScroll) ->
   link: (scope, elem, attrs) ->
     $rootScope.$on "toggleBodyScroll", ->
       if BodyScroll.disabled
-        elem.addClass "disable-scroll"
+        document.body.style.top = "-#{window.scrollY}px"
+        document.body.style.position = 'fixed'
+        document.body.style.overflowY = 'scroll'
       else
-        elem.removeClass "disable-scroll"
+        scrollY = parseInt(document.body.style.top)
+        document.body.style.position = ''
+        document.body.style.top = ''
+        document.body.style.overflowY = ''
+        window.scrollTo(0, scrollY * -1) if scrollY
```

Comment:
Is it appropriate to keep this code here? Or should I consider instead moving it to `cart_dropdown_controller` and remove the `body_scroll` directive? Also is it ok to use `document.body` straight up? Please let me know if there is any coffeescript styling related notes as well!

Reply:
- No strong opinion about keeping such a thing or move to `cart_dropdown_controller`, so why not keeping in here?
- I do not see any drawbacks about using `document.body` straight up, so perfect!

Anyway, thanks a lot ;)